{"name": "Проверка получения ошибки аутентификации курьера при вводе невалидных данных", "status": "passed", "description": "В тест по очереди передаются наборы данных с несуществующим логином или неверным паролем. Проверяются код и тело ответа.", "parameters": [{"name": "nonexistent_credentials", "value": "{'login': 'Boy.487', 'password': '#eXMySD7r4'}"}], "start": 1726654850559, "stop": 1726654850810, "uuid": "818f1de4-081a-4686-ad71-1d359c1dddf3", "historyId": "4434db397e8d886baadbf0c3107a97be", "testCaseId": "199be2f2af7455b14ed4795a4ee970da", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_login_nonexistent_data_not_found", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "minergenon"}, {"name": "thread", "value": "244-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}